.docker/gitpod/mathlib/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# This is the Dockerfile for `leanprover/mathlib:gitpod`.
+
+# gitpod doesn't support multiple FROM statements, (or rather, you can't copy from one to another)
```

Comment:
Can we use `FROM leanprover/lean` here to speed things up / use less space at gitpod?

Reply:
The problem is that gitpod logs you in as user `gitpod`, and there doesn't seem to be any way to change this, and I didn't like the idea of forcing the other containers to use that username too.

I guess it hadn't occurred to me to do global installs of the software, and then let `gitpod` add its own user...

Reply:
I've now tried this with both `leanprover/lean:alpine` and `leanprover/lean:debian`, and gitpod gets upset when it is adding its own special sauce layers on top of our container, with an error like:
```
build failed: cannot build workspace image: The command '/bin/bash -c /var/gitpod/layer.sh     && rm -f /var/gitpod/layer.sh' returned a non-zero code: 1.
```